website/docs/context/ref/Action.md

```diff
@@ -14,7 +14,7 @@ Accepts an optional `app` parameter in order to specify a specific app.
 
 ## Schema
 
-<https://fdc3.finos.org/schemas/next/context/action.schema.json> ([github](https://github.com/finos/FDC3/tree/main/schemas/context/action.schema.json))
+<https://fdc3.finos.org/schemas/2.1/context/action.schema.json> ([github](static/schemas/2.1/context/action.schema.json))
```

Comment:
This shouldn't have changed. The links should point to next in the unversioned docs or it'll point to the existing version's schema.

Reply:
I've fixed this by adjusting the generation